scipy/meson.build

```diff
@@ -16,7 +16,8 @@ if is_windows
     # https://bugs.python.org/issue28267
     bitness = run_command('_build_utils/gcc_build_bitness.py', check: true).stdout().strip()
     if bitness == '64'
-      add_project_arguments('-DMS_WIN64', language: ['c', 'cpp', 'fortran'])
+      # The `=` is a little odd, but avoids redefinition warnings (see meson#11592)
+      add_project_arguments('-DMS_WIN64=', language: ['c', 'cpp', 'fortran'])
```

Comment:
That is odd.  Can you explain why this works?  What is the difference in effect between `-DMS_WIN64` and `-DMS_WIN64=`?

Reply:
It's because CPython defined it like that, and then Meson changed its usage from the old to the new define here to avoid "`MS_WIN64` redefined" warnings, and now we're changing to match that for the same reason. I linked the relevant bug report in the code comment.

Reply:
Actually, is it possible to just delete this entirely now, since it is now handled by Meson itself in all versions of meson that SciPy supports as a build dependency?

Reply:
Good point, thanks @eli-schwartz. Done in gh-18411.